### PR TITLE
update deserialization errors tutorial inline with changes to ksqlDB master

### PR DESCRIPTION
**Pull request by big-andy-coates**
This includes changing the query we run against the processing logger to return something more useful, i.e. something that would allow the user to work out what the cause of the error was. This makes use of the new `encode` method to decode the base64 encoded record - nice!

cc @vpapavas as the author of the very useful `encode` UDF!

### CODE REVIEW

File: _includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc [hunk 1]
```diff
@@ -2,10 +2,17 @@ Now here comes the fun part. We know that at least one of the records produced h
...
 With the https://docs.confluent.io/current/ksql/docs/developer-guide/processing-log.html[KSQL Processing Log] feature enabled, you can query a stream called `KSQL_PROCESSING_LOG` to check for deserialization errors.
 
+The query below is extracting some of the data available in the processing log.
+As we configured the processing log to include the payload of the message, we can also use the `encode` method to convert the record from `base64` encoded into a human readable `utf8` encoding:
+
 +++++
 <pre class="snippet"><code class="sql">{% include_raw tutorials/deserialization-errors/ksql/code/tutorial-steps/dev/check-errors-query.sql %}</code></pre>
 +++++
 
+Notice we needed to quote the `topic` field in the `WHERE` clause, as its a reserved word.
```
Comment (colinhicks) [inline _includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc:12]:
```suggestion
Notice we needed to quote the `topic` field in the `WHERE` clause, as it's a reserved word.
```

File: _includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc [hunk 2]
```diff
@@ -2,10 +2,17 @@ Now here comes the fun part. We know that at least one of the records produced h
...
+
+While the `ERRORMESSAGE` is a little cryptic in this instances, the `CAUSE` and `MSG` columns would be enough to diagnose the issue here.
```
Comment (colinhicks) [inline _includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc:14]:
```suggestion
While the `ERRORMESSAGE` is a little cryptic in these instances, the `CAUSE` and `MSG` columns would be enough to diagnose the issue here.
```

File: _includes/tutorials/deserialization-errors/ksql/markup/dev/create-inputs.adoc [hunk 3]
```diff
@@ -1,8 +1,8 @@
 To start off the implementation of this scenario, you need to create a stream that represent sensors.
 This stream will contain a timestamp field called `TIMESTAMP` to indicate when the sensor was enabled.
-Each sensor will also have a field called `ENABLED` to indicate whether the status of the sensor.
-While the this stream acts upon data stored in a topic called `SENSORS_RAW`, we will create another stream called `SENSORS` to actually handle the sensors.
-This stream simply copies the data from the previous stream, but making sure that the `ID` field is used as key.
+Each sensor will also have a field called `ENABLED` to indicate the status of the sensor.
+While this stream acts upon data stored in a topic called `SENSORS_RAW`, we will create derived stream called `SENSORS` to actually handle the sensors.
+This stream simply copies the data from the previous stream, ensuring that the `ID` field is used as key.
```
Comment (colinhicks) [inline _includes/tutorials/deserialization-errors/ksql/markup/dev/create-inputs.adoc:5]:
```suggestion
This stream simply copies the data from the previous stream, ensuring that the `ID` field is used as the key.
```

File: _includes/tutorials/deserialization-errors/ksql/code/tutorial-steps/dev/check-errors-query.sql [hunk 4]
```diff
@@ -1,5 +1,7 @@
 SELECT
-    LOGGER,
-    LEVEL,
-    MESSAGE->DESERIALIZATIONERROR->ERRORMESSAGE
-FROM KSQL_PROCESSING_LOG EMIT CHANGES LIMIT 1;
+    message->deserializationError->errorMessage,
+    encode(message->deserializationError->RECORDB64, 'base64', 'utf8') AS MSG,
```
Comment (colinhicks) [inline _includes/tutorials/deserialization-errors/ksql/code/tutorial-steps/dev/check-errors-query.sql:3]: Nice :)

File: _includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc (hunk 2 above)
Comment (big-andy-coates) [reply _includes/tutorials/deserialization-errors/ksql/markup/dev/check-errors-query.adoc]: There's only a single error in the example, so went with `this instance`.

**Pull request merged**